cardano-node/README.md

```diff
@@ -1,3 +1,103 @@
+  networking, with configuration, CLI, logging and monitoring.
+
+- The node no longer incorporates wallet or explorer functionality. The wallet
+  backend and explorer backend are separate components that run in separate
+  external processes that communicate with the node via local IPC.
+
+## How to build
+
+### Stack
+Use [Haskell Stack - Version 2.1.3](https://haskellstack.org/) to build this project:
+
+```
+$ cd cardano-node
+$ stack build
+```
+
+## Cabal
+
+Use [Cabal - Version 2.4.1.0](https://www.haskell.org/cabal/) to build this project:
+
+```
+$ cd cardano-node
+$ cabal new-build
+```
+
+## Cardano Node CLI
+
+This refers to the client that is used for running a node.
+
+The general synopsis is as follows:
+```
+
+Usage: cardano-node --topology FILEPATH --database-path FILEPATH
+                    --genesis-file FILEPATH [--delegation-certificate FILEPATH]
+                    [--signing-key FILEPATH] --socket-dir FILEPATH
+                    [--host-addr HOST-NAME] --port PORT
+                    --config NODE-CONFIGURATION [--help] [--help-tracing]
+                    [--help-advanced]
+  Start node of the Cardano blockchain.
+```
+`--topology` - Filepath to a topology file describing which peers the node should connect to.
+
+`--database-path` - Path to the blockchain database.
+
+`--genesis-file` - Path to the genesis file of the chain you are connecting to.
+
+`--delegation-certificate` - Optional path to the delegation certificate. The delegation certificate allows the delegator (the issuer of said certificate) to give his/her own block signing rights to somebody else (the delegatee). The delegatee can then sign blocks on behalf of the delegator.
+
+`--signing-key` - Optional path to the signing key.
+
+`--socket-dir` - Path to the socket directory.
+
+`--host-addr` - Optionally specify your node's IPv4 or IPv6 address.
+
+`--port` - Specify which port to assign to the node.
+
+`--config` - Specify the filepath to the config `.yaml` file. This file is responsible for all the other node's required settings. See examples in `configuration` (e.g. `log-config-0.yaml`).
+
+
+## Configuration `.yaml` files
+
+The `--config` flag points to a `.yaml` file that is responsible to configuring the logging & other important settings for the node.
+Some of the more important settings are as follows:
+
+`NodeId: 0`  -- Soon to be removed from config file. Used to identify nodes in local clusters.
+
+`Protocol: RealPBFT` -- Protocol the node will execute
+
+`RequiresNetworkMagic`: RequiresNoMagic -- Used to distinguish between mainnet (`RequiresNoMagic`) and test nets (`RequiresMagic`)
+
+`ViewMode: SimpleView` -- Choose between SimpleView or LiveView
+
+
+## Logging
+
+Logs are output to the `log/` dir.
+
+
+## Connect to mainnet proxy
+
+Run `./scripts/mainnet-proxy-follower.sh`
+
+This script connects to an IOHK maintained [byron proxy](https://github.com/input-output-hk/cardano-byron-proxy) which is following mainnet/
+
+
```

Comment:
Check with @disassembler. I think we now have new nodes running on mainnet. As soon as we do, we can update the instructions here to connect to mainnet using the new nodes on mainnet.